OpenXmlPowerTools/MetricsGetter.cs

```diff
@@ -39,6 +40,17 @@ public class MetricsGetterSettings
 
     public class MetricsGetter
     {
+        protected static Lazy<Graphics> Graphics {
```

Comment:
Does this need to be `protected` instead of `private`? Also, make it `readonly`

Reply:
I made it private, but no need to make it readonly as it only has a getter.

Reply:
Oh I didn't see that. You're actually creating a new lazy each time :) Instead, do this:

```csharp
private static Lazy<Graphics> Graphics { get; } = new Lazy<Graphics>(()=>...)
```

Reply:
Haha ! Good catch